css/pages/index.css

```diff
@@ -3,50 +3,122 @@
 }
 
 .container {
-  width: 750px;
+  width: 100%;
   text-align: center;
   font-size: 1.5rem;
 }
 
+.second-screen {
+  width: 100%;
+}
+
 .cards-title {
-  margin-bottom: 32px;
-  color: #e20665;
+  margin-top: 1041px;
+  margin-bottom: 60px;
```

Comment:
use shorthand for margin :
refer : [https://developer.mozilla.org/en-US/docs/Web/CSS/margin](url)

Reply:
Yes will do that!